source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -2061,8 +2070,11 @@ AdaptiveNamespaceStart
         [propget] HRESULT Id([out, retval] HSTRING* value);
         [propput] HRESULT Id([in] HSTRING value);
 
-        [propget] HRESULT IconUrl([out, retval] Windows.Foundation.Uri** value);
-        [propput] HRESULT IconUrl([in] Windows.Foundation.Uri* value);
+        [propget] HRESULT IconUri([out, retval] HSTRING* value);
+        [propput] HRESULT IconUri([in] HSTRING value);
+
+        [propget] HRESULT IsIconUriRelative([out, retval] boolean* value);
+        [propput] HRESULT IsIconUriRelative([in] boolean value);
```

Comment:
I don't think we want a putter for this.

Reply:
(for all isrelative properties)

---
In reply to: [187156141](https://github.com/Microsoft/AdaptiveCards/pull/1437#discussion_r187156141) [](ancestors = 187156141)